fvm/src/call_manager/default.rs

```diff
@@ -110,12 +138,19 @@ where
         res
     }
 
-    fn finish(mut self) -> (i64, Backtrace, Self::Machine) {
+    fn finish(mut self) -> (FinishResult, Self::Machine) {
```

Comment:
nit: `FinishResult` sounds like a specialized `Result` type. Can we change this type name to `FinishRet`, parallel with `ApplyRet`?